scripts/dockerfiles/runtime/Dockerfile

```diff
@@ -48,6 +48,25 @@ ADD ecs /ecs/
 COPY entrypoint.sh /entrypoint.sh
 RUN chmod +x /entrypoint.sh
 
+ARG FLB_VERSION
+ARG AWS_FOR_FLUENT_BIT_VERSION
+ARG OS_PRETTY_NAME
+
+LABEL author="FireLens Team <aws-firelens@amazon.com>" \
+      vendor="Amazon Web Services" \
+      version=${AWS_FOR_FLUENT_BIT_VERSION} \
+      com.aws-for-fluent-bit.fluent-bit.version=${FLB_VERSION} \
+      com.aws-for-fluent-bit.os.version=${OS_PRETTY_NAME} \
+      org.opencontainers.image.title="AWS for Fluent Bit" \
+      org.opencontainers.image.authors="FireLens Team <aws-firelens@amazon.com>" \
+      org.opencontainers.image.description="Fluent Bit is an open source and multi-platform Log Processor and Forwarder which allows you to collect data/logs from different sources, unify and send them to multiple destinations. It's fully compatible with Docker and Kubernetes environments." \
```

Comment:
Should we modify this description to talk about AWS for Fluent Bit rather than Fluent Bit? Something along the lines of optimized to run on AWS environments / AWS container orchestration envs like ECS/EKS.

Thoughts?